### ISSUE TITLE
allow to signin visitor without site.domain

### ISSUE THREAD

**Issue by elbidone** [meumobi/sitebuilder#2]
we'd like to offer a generic infobox app, available from Apps Stores. 
Each company will create its own App from site-builder
The visitor is invited from site-builder to download InfoBox App from Apps Store and signin with an auto-generated password and mail.
That means that on 1st connection to /api the app doesn't know the site.domain. Site-builder should be able to find out corresponding app for determined visitor.mail.

**Comment by elbidone** [comment]
Raises an error on INT

````<br />
<b>Notice</b>:  Trying to get property of non-object in <b>/home/meumobi/PROJECTS/int-meumobi.com/releases/20150430191655/sitebuilder/app/controllers/api/ApiController.php</b> on line <b>198</b><br />
{"success":true,"token":"6b56dd98c1845a1e52b3fc5cf57fb8d631ae3948","visitor":{"first_name":"Victor","last_name":"Dias","email":"victor.dias@meumobi.com","site":"infobox.int-meumobi.com"}}
```

**Comment by juliogreff** [comment]
Fixed by 9861e8d4710abe1e68f017016041a7cd0b6f565b
